### PR TITLE
Addy Abid - String Manipulation 2

### PR DESCRIPTION
**Pull request by AddyAbid**
_Users_addyabid_repos_c1021-code-solutions_string-manipulation-2_index html

### CODE REVIEW

File: string-manipulation-2/src/capitalize-words.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+create new storage for output and assign the new value of the word joined back together at each space
+return output
+*/
+
+function capitalizeWords(string) {
+  var word = string.toLowerCase();
+  word = word.split(' ');
+  for (var i = 0; i < word.length; i++) {
+    word[i] = word[i][0].toUpperCase() + word[i].substring(1);
+    var outPut = word.join(' ');
```
Comment (codingcodymiller) [inline string-manipulation-2/src/capitalize-words.js:16]: You should do this after the loop has ended

File: string-manipulation-2/src/capitalize-word.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+return storage
+*/
+
+function capitalizeWord(word) {
+  var lower = word.toLowerCase();
+  var upper = word[0].toUpperCase();
+  var outPut = upper + lower.substr(1);
+  if (outPut === 'Javascript') {
+    outPut = 'JavaScript';
+  }
```
Comment (codingcodymiller) [inline string-manipulation-2/src/capitalize-word.js:17]: You should check for this before you do any other work, so you don't do things you don't end up using

File: string-manipulation-2/src/num-vowels.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+add each vowel to storage count
+return storage
+
+*/
+
+function numVowels(string) {
+  var outPut = 0;
+  for (var i = 0; i < string.length; i++) {
+    if (string[i] === 'a' || string[i] === 'A' || string[i] === 'e' || string[i] === 'E' || string[i] === 'i' || string[i] === 'I' || string[i] === 'o' || string[i] === 'O' || string[i] === 'u' || string[i] === 'U') {
+      outPut += string[i].length;
```
Comment (codingcodymiller) [inline string-manipulation-2/src/num-vowels.js:15]: Why the length? You should just add `1`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AddyAbid** [comment]
_Users_addyabid_repos_c1021-code-solutions_string-manipulation-2_index html (1)

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**